libnmstate/__init__.py

```diff
@@ -36,3 +41,9 @@
     "schema",
     "PrettyState",
 ]
+
+
+def __version__():
```

Comment:
iThis should be a property called `__version__` not a method....

Reply:
Done!